source/guides/administrator.rst

```diff
@@ -57,7 +58,6 @@ Learn how to host Mattermost to meet your networking requirements.
 
    /deployment/deployment.md 
    /deployment/desktop-app-deployment*
-   AppConfig for EMM Solutions </deployment/mobile-appconfig.rst>
```

Comment:
@amyblais why did we remove this one?

Reply:
Broken link, changed it.
It's in the Mobile section too.